src/backend/vulkano.rs

```diff
@@ -550,16 +550,16 @@ impl Renderer {
                     //
                     // Texture coordinates range:
                     // - left to right: 0.0 to 1.0
-                    // - bottom to top: 1.0 to 0.0
+                    // - bottom to top: 0.0 to 0.1

```

Comment:
@Kurble 0.0 to 1.0, or?

Reply:
Totally, nice find :)
I still need to take some time to write out the example anyway